services/ses/index.js

```diff
@@ -17,6 +17,11 @@ function filterInternalErrors(error) {
   throw error;
 }
 
+function validateContact(contact) {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // eslint-disable-line max-len
```

Comment:
This is pretty grim and probably incorrect. I'd use something simpler like `/.+@.\..+/`